proposals/0000-new-visible-forall.rst

```diff
@@ -0,0 +1,126 @@
+New syntax for Visible ForAll
+=============================
+
+.. author:: Viktor WW
+.. date-accepted::
+.. ticket-url:: 
+.. implemented::
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/729>`_.
+.. sectnum::
+.. contents::
+
+
+This proposal introduces a syntax for visible forall type variables.
+
+Motivation
+----------
+
+The Visible ForAll awesome feature was added to GHC as an extension with ... a temporary syntax.
```

Comment:
As far as I'm aware the current visible type argument syntax is not temporary. Where is it described as temporary?

Reply:
Can you point to a discussion where the committee said that this is a tentative syntax that might revised later? I don’t think it is.